spec/manageiq_performance/configuration_spec.rb

```diff
@@ -207,90 +195,90 @@
     end
 
     it "defines ManageIQPerformance.config.default_dir" do
-      expect(ManageIQPerformance.config.default_dir).to eq("/tmp/miq_perf")
-      expect(ManageIQPerformance.config["default_dir"]).to eq("/tmp/miq_perf")
+      expect(ManageIQPerformance.config.default_dir).to eq "/tmp/miq_perf"
+      expect(ManageIQPerformance.config["default_dir"]).to eq "/tmp/miq_perf"
     end
 
     it "defines ManageIQPerformance.config.log_dir" do
-      expect(ManageIQPerformance.config.log_dir).to eq("tmp/my_log_dir")
-      expect(ManageIQPerformance.config["log_dir"]).to eq("tmp/my_log_dir")
+      expect(ManageIQPerformance.config.log_dir).to eq "tmp/my_log_dir"
+      expect(ManageIQPerformance.config["log_dir"]).to eq "tmp/my_log_dir"
```

Comment:
did you want to pull these reformats out of here to make this PR smaller and to the point?
(either way is good for me)